src/actions/article.js

```diff
@@ -11,8 +9,14 @@ export const viewArticle = (content) => { return { type: VIEW_ARTICLE, content }
 
 export const getArticle = (bucketId, itemId) => {
```

Comment:
ideally we should be fetching the article/package based on the `item.url` not constructing it manually.